Source/Evolve/WarpXEvolve.cpp

```diff
@@ -1010,3 +982,13 @@ WarpX::VayDeposition ()
     "WarpX::CurrentCorrection: requires WarpX build with spectral solver support.");
 #endif
 }
+
+// Execute Python callbacks of the type given by the input string
```

Comment:
Thanks for implementing this. Though, could this routine be put elsewhere, perhaps in `WarpX_py.H`, since that is where `warpx_callback_py_map` is defined? I don't think that this needs to be a method in the WarpX class.

Reply:
I agree that is a much better place to put it.